manuscript.md

```diff
@@ -31,7 +31,7 @@ OSS (open-source software) refers to software where the source code—the core i
 Unlike "closed-source" or "proprietary" software, where the code remains private, OSS provides transparency by design, which is essential for fostering trust, collaboration, and improvement in science.
 However, the open- or closed-source label simply addresses code visibility; it's the software license that determines who can use, modify, or distribute the software and under what conditions.
 
-A common misconception is that closed-source software with "academic" licenses are equivalent to open-source. Adding to the confusion, some journals now require code availability as a condition of publication. This has led researchers to upload their closed-source software to collaborative platforms. While this provides the appearance of openness through features like issue trackers, the source code remains locked away. Even if well-intentioned, this can be misleading to other scientists and in our view discouraged.
+A common misconception is that closed-source software with "academic" licenses are equivalent to open-source. Adding to the confusion, some journals now require code availability as a condition of publication. This has led researchers to upload their closed-source software to collaborative platforms. While this provides the appearance of openness through features like issue trackers, the source code remains locked away. Even if well-intentioned, this can be misleading to other scientists and in our view discouraged. [what about github repos starting as open-source, changing over time to be closed source under the same name and caursing confusion. Or being open-source at the time of review and removing repositories. I think this should maybe be mentioned here? On second though, probably not here, but I think we should make this point later on.]
```

Comment:
@RobbinBouwmeester I agree with you, that is another problem, can you highlight that? It is true that some software starts from open-source and ends up in a company with the same software through a route of close, source, change license, and finally making the software commercial. 

Please feel free to include that as another point.

Reply:
I agree too - just be cautious not to call out specific examples here. We're after all not trying to shame existing tools, but rather help us build a better future 😄